src/main/java/com/entropyteam/entropay/employees/common/CrudService.java

```diff
@@ -3,15 +3,15 @@
 import java.util.List;
 import java.util.Optional;
 
-public interface CrudService<T, K> {
+public interface CrudService<DTO, Key> {
```

Comment:
I'd move this interface to something like com.entropyteam.entropay.common.services, in order to keep it outside employees or any other module-specific package.